Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use fixed snapshot of Debian to create a deterministic environment.
 # Snapshot tags can be found at https://hub.docker.com/r/debian/snapshot/tags
-ARG debian_snapshot=buster-20200327
+ARG debian_snapshot=buster-20200720
```

Comment:
After some research: It does rebuild the whole image if the docker back end is changed (e.g. to BuildKit), but the pull of the original image fails, partially because it doesnt exist any more (so it seems), so I changed that parameter to the currently available image from Docker.io. It looks like the gcb infrastructure hasnt got that image available either, and therfore it fails. BuildKit builds do seem to work (I have it enabled by default on my x86 and the packet machine), but for some reason the CI here has a kink in it...